src/Gateway/PayplugGateway.php

```diff
@@ -1261,18 +1261,24 @@ public function get_current_mode()
      */
     public function get_api_key($mode = 'test')
     {
+		$key = $this->get_option('payplug_' . $mode . '_key');
+		$jwt = $this->get_option('jwt');
```

Comment:
in order to preserve retro compatibility, the jwt token should be kept under the same option name 'payplug_test_key' and 'payplug_live_key'